### PR TITLE
More infomation in TNT logging

### PR DESCRIPTION
**Pull request by X-Green**
With gametime and order-in-gametick of explosions shown

### CODE REVIEW

**Comment by gnembon** [comment]
That's a good idea to add tick time. Isn't the order information little redundant? (messages will be showing up in order anyways from 0 to <whatever>

**Comment by X-Green** [comment]
hmmm...
Maybe orders should be hidden when there's only single explosion in one tick?
Or should all of them be hidden in "biref" mode, thus shown in "full" mode?

File: src/main/java/carpet/logging/logHelpers/TNTLogHelper.java [hunk 1]
```diff
@@ -24,18 +26,30 @@ public void onPrimed(double x, double y, double z, Vec3d motion)
...
+            tntCount = 0;
+            lastGametime = gametime;
+        }
+        tntCount++;
         LoggerRegistry.getLogger("tnt").log( (option) -> {
             switch (option)
             {
                 case "brief":
                     return new BaseText[]{Messenger.c(
+                            "g  #" + String.valueOf(tntCount),
```
Comment (gnembon) [inline src/main/java/carpet/logging/logHelpers/TNTLogHelper.java:41]: yeah, if you could just remove this one line from brief that would be better imo

**Comment by X-Green** [comment]
Yeah

**Pull request merged**